phoenix-core/src/main/java/org/apache/phoenix/iterate/RegionScannerFactory.java

```diff
@@ -285,8 +286,11 @@ public boolean nextRaw(List<Cell> result) throws IOException {
             }
           }
           Cell arrayElementCell = null;
-          if (arrayFuncRefs != null && arrayFuncRefs.length > 0 && arrayKVRefs.size() > 0) {
-            int arrayElementCellPosition = replaceArrayIndexElement(arrayKVRefs, arrayFuncRefs, result);
+          if (serverParsedFuncRefs != null && serverParsedFuncRefs.length > 0 && serverParsedKVRefs.size() > 0) {
+            int
```

Comment:
The formatting seems a little messed up here

Reply:
actually I used IntelliJ formatting to avoid running longer than 100 or 150 chars in a line. I can split into 2 line instead of 3. just went with the default formatting